installconfig.adoc

```diff
@@ -502,7 +503,7 @@ installation.
 3+^|*zimbra-imapd*
 .1+.^|mailbox server or +
 independent server
-| Add to upstream IMAP Servers?: | default `no`.  If `yes`, the
+| Add to upstream IMAP Servers? | default `no`.  If `yes`, the
```

Comment:
Again, consistency seems to indicate we use *no* and *yes* (bold) instead of (code) format.